<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WarningHistory</class>
 <widget class="QDialog" name="WarningHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
border-radius:12px;
background-color: rgb(80, 80, 80);
font-size:22px;
color:rgb(255, 255, 0);
}

QPushButton::pressed{
background-color: rgb(120, 120, 120);
}</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Mot_pb">
   <property name="geometry">
    <rect>
     <x>749</x>
     <y>385</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返
回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lastday">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>前一日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nextday">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>后一日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteall">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>290</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>删
除
所
有</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletethisday">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>195</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>删
除
当
日</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_today">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>今日</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>721</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px;
border:1px solid gray;</string>
   </property>
   <property name="text">
    <string>0/0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pagedown">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>100</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>下
一
页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pageup">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>5</y>
     <width>51</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
</string>
   </property>
   <property name="text">
    <string>上
一
页</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
